src/Component/Homepage.js

```diff
@@ -0,0 +1,217 @@
+      .then((response) => response.json())
+      .then(async (response) => {
+        localStorage.setItem('first_name', response.given_name);
+        localStorage.setItem('last_name', response.family_name);
+        localStorage.setItem('email', response.email);
+
+        data = new FormData();
+        data.append('first_name', localStorage.getItem('first_name'));
+        data.append('last_name', localStorage.getItem('last_name'));
+        data.append('username', localStorage.getItem('first_name'));
+        data.append('email', localStorage.getItem('email'));
+
+        config = {
+          method: 'post',
+          url: 'http://127.0.0.1:8000/employees/employees/',
+          headers: {
+            'Content-Type': 'application/x-www-form-urlencoded',
+          },
+          data,
+        };
+
+        await axios(config)
+          .then(async (res) => {
+            localStorage.setItem('user_id', res.data.detail.id);
+            localStorage.setItem('django_Token', res.data.Token);
+          })
+          .catch(() => {
+            navigate('/');
+          });
+      });
+  }
+
+  // After Successfully Login
+  const onSuccess = async (res) => {
+    const Auth = res.code;
+    await GenerateRefreshToken(Auth);
+
+    navigate('/dashboard');
+  };
+
+  const onFailure = () => {};
+
+  useEffect(() => {
+    const Refresh_Token = localStorage.getItem('Refresh_Token');
+    if (Refresh_Token == null) {
+      return;
+    }
+
+    let chk = true;
+    const Sec = new Date().getTime() - localStorage.getItem('Gen_Date');
+
+    if (Sec >= 3500000) {
+      chk = false;
+    }
+
+    if (!chk) {
+      GenerateAccesstoken(Refresh_Token);
+    }
+
+    navigate('/Dashboard');
+  });
+
+  const theme = createTheme();
+
+  return (
+    <div>
+      <ThemeProvider theme={theme}>
+        <Grid
+          container
+          component="main"
+          sx={{
+            height: '100vh',
+            width: '100%',
+            backgroundColor: '#feeae6',
+          }}
+        >
+          <CssBaseline />
+
+          <Grid
+            item
+            xs={false}
+            sm={4}
+            md={7}
+            sx={{
+              display: 'flex',
+              backgroundRepeat: 'no-repeat',
+              backgroundColor: '#feeae6',
+              backgroundSize: 'cover',
+              backgroundPosition: 'center',
+            }}
+          >
+            <div sx={{ display: 'flex', justifycontent: 'left' }}>
+              <img src={Image1} alt="Hurre" height="570" width="1000" />
+            </div>
+            <Box
+              sx={{
+                my: 8,
+                mx: 4,
+                display: 'flex',
+                flexDirection: 'column',
+                alignItems: 'center',
+              }}
+            >
+              <img src={Squad} alt="Squadlogo" />
+              <Typography
+                component="h1"
+                variant="h5"
+                style={{ 'text-align': 'center' }}
```

Comment:
Let's avoid inline styles?

Reply:
I tried to make it from css file but didn't applied.